samples/algorithms/repeat-until-success/RepeatUntilSuccess.qs

```diff
@@ -0,0 +1,110 @@
+namespace Microsoft.Quantum.Samples.RepeatUntilSuccess {
```

Comment:
We should make sure this has an appropriate copyright header; should be able to use one from similar files elsewhere in the samples.

Reply:
will do!